projects/addon-editor/directives/tiptap-editor/tiptap-editor.directive.ts

```diff
@@ -35,8 +36,23 @@ export class TuiTiptapEditorDirective {
     }
 
     @Output()
-    readonly valueChange = this.editor.valueChange$;
+    readonly valueChange = this.editor.valueChange$.pipe(
+        switchMap((value, index) =>
+            /**
+             * @note:
+             * for the initial setting of the value,
+             * we shouldn't make a delay so that the changes have time to take effect
+             */
+            index && !!value ? timer(300).pipe(map(() => value)) : of(value),
```

Comment:
`!!` is unnecessary. Is 300 an empirically figured number? Does it depend on CPU speed?